martypy/Marty.py

```diff
@@ -82,6 +85,13 @@ class Marty(object):
     ADD_ON_TYPE_NAMES = [
         "IRFoot"
     ]
+    class LED(Enum):
```

Comment:
Would `LedGroup` or `DiscoGroup` or `LedKind` or `DiscoType` or some other permutation be a better name? :thinking:

Reply:
My thinking was that having a short name would be easier for users to type in, so that when theyre passing it in its marty.LED.ARMS instead of lets say marty.LedGroup.ARMS. But do you think its better to have a more explicit class name for the sake of whoevers reading the code?

Reply:
Hmmm, I agree it is good to keep it short. I was worried that LED might get confused with the LEDs on the back of Marty's head. Maybe just `Disco` then? It would kind of read like English (`Marty.Disco.ARMS`) which is usually a good sign.